app/cafe.py

```
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (OutdatedVaccineError,
+                        NotWearingMaskError,
+                        NotVaccinatedError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("NotVaccinatedError")
+
+        visitor_date = visitor["vaccine"]["expiration_date"]
+
+        if visitor_date < datetime.date.today():
+            raise OutdatedVaccineError("OutdatedVaccineError")
+        if ("wearing_a_mask" in visitor.keys()
+                and visitor["wearing_a_mask"] is False):
+            raise NotWearingMaskError("NotWearingMaskError")
```

Issue: The current logic only raises `NotWearingMaskError` if the key `'wearing_a_mask'` exists and is `False`. According to the requirements, you must raise `NotWearingMaskError` if the key is missing or its value is `False`. Please update this check to raise the error if the key is missing or its value is not `True`.

Checklist reference: 'Raise NotWearingMaskError if the visitor is not wearing a mask (either the key is missing or its value is False).'